Sources/SwiftInspectorCommands/InitializerCommand.swift

```diff
@@ -0,0 +1,84 @@
+// Created by Francisco Diaz on 3/27/20.
+//
+// Copyright (c) 2020 Francisco Diaz
+//
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import ArgumentParser
+import Foundation
+import SwiftInspectorCore
+
+final class InitializerCommand: ParsableCommand {
+  static var configuration = CommandConfiguration(
+    commandName: "initializer",
+    abstract: "Finds information about the initializers of the specified type"
+  )
+
+  @Option(help: "The absolute path to the file to inspect")
+  var path: String
+
+  @Option(help: "The name of the type to look information of the initializer")
```

Comment:
Let's reword this sentence. It's slightly off as-is.

Reply:
https://github.com/fdiaz/SwiftInspector/pull/15/commits/21d3b9d79d11f6aa134bf13a106ab4fc20f77262